db/seeds.rb

```diff
@@ -6,14 +6,24 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
+require 'find'
+
 # Initialising database with service types from Catalog repo
 if (Dir["mastermind-services"]).length == 0 then
   `git clone https://github.com/martel-innovate/MasterMind-Service-Catalog mastermind-services`
 end
-for path in Dir['mastermind-services/*/*/']
-  mastermindConf = YAML::load(File.open(path+'mastermind.yml'))
-  dockerCompose = YAML::load(File.open(path+'docker-compose.yml'))
-  ServiceType.create(name: mastermindConf["name"], description: mastermindConf["description"], version: mastermindConf["version"], service_protocol_type: mastermindConf["protocol_type"], ngsi_version: mastermindConf["ngsi_version"], configuration_template: File.read(path+'mastermind.yml'), deploy_template: File.read(path+'docker-compose.yml'))
+#for path in Dir['mastermind-services/*/*/']
+#  mastermindConf = YAML::load(File.open(path+'mastermind.yml'))
+#  dockerCompose = YAML::load(File.open(path+'docker-compose.yml'))
+#  ServiceType.create(name: mastermindConf["name"], description: mastermindConf["description"], version: mastermindConf["version"], service_protocol_type: mastermindConf["protocol_type"], ngsi_version: mastermindConf["ngsi_version"], configuration_template: File.read(path+'mastermind.yml'), deploy_template: File.read(path+'docker-compose.yml'))
+#end
+Find.find('mastermind-services') do |path|
```

Comment:
Mm, why do we have this repetition here as well?

Reply:
@taliaga It was just the older way of doing it, but I removed the commented lines since they aren't needed anymore